### ISSUE TITLE
Overview page ally issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#11406]
### Description:

![image](https://user-images.githubusercontent.com/20179540/124704441-b770e780-df11-11eb-91f5-cda732c15110.png)
Rating functionality is not accessible through keyboard 

![image](https://user-images.githubusercontent.com/20179540/124704520-d96a6a00-df11-11eb-9303-35084c727b74.png)
SDK download buttons need descriptive ally text

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---
